contenttype.go

```diff
@@ -300,6 +300,19 @@ func (mediaType *MediaType) String() string {
 	return stringBuilder.String()
 }
 
+// Mime returns the MIME type without any of the parameters
+func (mediaType MediaType) Mime() string {
```

Comment:
Golang naming convention states that acronyms must be capitalized (for example https://pkg.go.dev/net/http#ParseHTTPVersion) so this should be named MIME.

Reply:
True, i'll change that.

Reply:
Thanks!